plugins/inara.py

```diff
@@ -411,23 +419,17 @@ def journal_entry(cmdr, is_beta, system, station, entry, state):
             # if our cargo differers from last we checked, we're at a station,
             # and our flood limit isnt covered, queue an update
```

Comment:
differ**ER**s ?

isnt ?

Reply:
That code should have been removed anyway :D